bmrg-flow/templates/deployment-service.yaml

```diff
@@ -97,6 +97,15 @@ spec:
             mountPath: /opt/newrelic
           {{- end }}
           {{- if and (eq $k "controller") (eq $values.workers.logging.type "elastic" ) }}
+          {{- if $.Values.database.mongodb.tls.enable }}
```

Comment:
Can't we use a single statement with an or for the wrapper and then adjust what files are mounted?

Reply:
Not really,  we need to know whether we need to import certificates for one or the other or both and it has to be done in single block given the java service is started after the keystore imports.

Reply:
Well in the case the IF statement should be combined and then wrap some of the command line with individual IF additions.

Reply:
Actually yes, this one can be combined.

Reply:
Im not seeing the change in the above code